tests/test_qiskit_device.py

```diff
@@ -1,14 +1,32 @@
+# Copyright 2021-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+This module contains tests qiskit devices for PennyLane IBMQ devices.
+"""
+from unittest.mock import Mock
 import numpy as np
 import pytest
 
+from qiskit_aer import noise
+from qiskit.providers import BackendV1, BackendV2
+from qiskit_ibm_runtime.fake_provider import FakeManila, FakeManilaV2
+
 import pennylane as qml
 from pennylane_qiskit import AerDevice
 from pennylane_qiskit.qiskit_device import QiskitDevice
-from qiskit_aer import noise
-from qiskit.providers import BackendV1, BackendV2 
-from qiskit_ibm_runtime.fake_provider import FakeManila, FakeManilaV2
-from unittest.mock import Mock
-from qiskit_ibm_runtime.options import Options
```

Comment:
Not needed anymore?

Reply:
Yep! They were not being used in any tests.